src/System.Threading.Channels/ref/Configurations.props

```diff
@@ -1,8 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
+      netstandard1.3;
```

Comment:
Since there aren't any code changes, that suggests it "just works" on 1.3 (which makes sense, since that's what it was targeting in corefxlab).  At some point we will depend on newer APIs, but in the meantime, is there a need to keep the 2.0 build, or can we just have the 1.3 build?

Reply:
I highly recommend keeping the netstandard20 build as it does add the value of removing package dependencies for folks that are running on NS20 compatible platforms.

Reply:
Ok